src/options.coffee

```diff
@@ -131,4 +131,32 @@ options =
   version:
     description: "Show version number.\n"
 
+  'hook-worker-timeout':
```

Comment:
Cool, I'm not totally sure what a majority of the variables mean, I can look into it later and ask for specific ones. If there are any other names as well that'd make more sense just let me know!